main_server/local_server/server_admin/server_admin_javascript.js

```diff
@@ -1392,7 +1392,22 @@ function BMLT_Server_Admin()
         save_button.innerHTML = g_Save_meeting_button_name;
         this.validateMeetingEditorButton(in_meeting_id);
     };
-    
+
+    this.alertIssue = function ( message ) {
+        alert(message);
+    };
+
+    this.validateSave = function ( in_meeting_id ) {
+        var virtual_meeting_link = document.getElementById('bmlt_admin_single_meeting_editor_' + in_meeting_id + '_meeting_virtual_meeting_link_text_input');
+        var virtual_meeting_additional_info = document.getElementById('bmlt_admin_single_meeting_editor_' + in_meeting_id + '_meeting_virtual_meeting_additional_info_text_input');
+        if (virtual_meeting_link.value !== null && virtual_meeting_additional_info.value == null) {
+            alert("Virtual Meeting Additional Information field must be set.");
```

Comment:
I'm not sure the additional info field should be required, but I do think it should be _strongly suggested_. Thoughts?

Reply:
I don't think it should be required either but I barely care so could go either way :)

Reply:
Ok, this was an attempt at an implementation of this ticket.  How did you envision it would be?  https://github.com/bmlt-enabled/bmlt-root-server/issues/358.

Reply:
The ticket says "we should ask them if they're sure, and explain why it's a good idea to at a minimum put the Zoom ID, password, or similar information in that field. It's fine to call out zoom specifically, let's be practical."

"We should ask them if they are sure"

If you guys both feel strongly that it should be required, then I'm fine with that - the UI should indicate clearly _why_ it is required, though.